pkg/pykernel/runtime/bootstrap.py

```
@@ -0,0 +1,275 @@
+def _worker():
+    global _worker_tid
+    _worker_tid = threading.get_ident()
+    while True:
+        item = _exec_queue.get()
+        if item is None:
+            return
+        exec_id, code, output_cap = item
+        _run_code(exec_id, code, output_cap)
+
+
+def _cancel_worker():
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_cancel_worker`

Add return type annotation: `None`

(ANN202)